### PR TITLE
Creating video from image

### PR DESCRIPTION
**Pull request by bartoszrud**
Addressing #27

### CODE REVIEW

**Comment by BartWojtowicz** [comment]
Looking good, nice work 👍🏻 Once the conflicts are resolved, I can approve 😊

File: tests/test_video.py [hunk 1]
```diff
@@ -88,6 +94,17 @@ def test_can_be_downsampled_to(
...
 
+def test_video_from_image():
+    with Image.open(_Configuration.SMALL_IMG_PATH) as img:
+        img = np.array(img)
+    video = Video.from_image(img, fps=30, length=0.5)
+
+    assert video.frames.shape == (15, *img.shape)
+    assert video.fps == 30
+    assert np.array_equal(video.frames[0], img)
+    assert np.array_equal(video.frames[-1], img)
```
Comment (BartWojtowicz) [inline tests/test_video.py:105]: Can we also visually confirm that the generated video is correct? I remember having issues with PIL flipping the axis so I would make sure by saving the video and confirming it looks correct.

Comment (bartoszrud) [reply tests/test_video.py]: After small fix it is correct.

**Review by BartWojtowicz — APPROVED** [review]
LGTM 😎

**Pull request merged**